Sources/XCTestDynamicOverlay/XCTIsTesting.swift

```diff
@@ -1,6 +1,8 @@
 import Foundation
 
-#if !os(WASI)
+#if RELEASE || os(WASI)
```

Comment:
@mbrandonw Had this staged from an earlier pairing session. Should we go forward with it?